modules/netweaver/variables.tf

```diff
@@ -109,10 +109,6 @@ variable "device_3" {
   default     = "swap"
 }
 
-variable "startup_script" {
```

Comment:
The CHANGELOG needs to indicate that variables are being removed.

Reply:
added that comment in CHANGELOG.md file